artprima/prometheus-metrics-bundle/1.0/config/packages/test/prometheus_metrics.yaml

```diff
@@ -0,0 +1,6 @@
+artprima_prometheus_metrics:
+    namespace: myapp
+    type: in_memory
+    ignored_routes:
```

Comment:
You do not need to redeclare ignored_routes or namespace. They will be set from the `packages/promethues_metrics.yaml`.